winter/web/autodiscovery.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+
+from winter.core import Component
+from winter.core.module_discovery import get_all_classes
+from winter.core.module_discovery import import_recursively
+from winter.web.routing import Route
+from winter.web.routing import get_route
+
+
+def get_routes_by_package(package_name: str) -> List[Route]:
```

Comment:
I would rename to `get_routes_for_package` to be consistent with `create_django_urls_for_package`
Or shorter versions: `find_package_routes`/`get_package_routes`

(not a blocker)